mbtiles/__init__.py

```diff
@@ -1,62 +1,8 @@
-import sys
+"""rio-mbtiles package"""
```

Comment:
We're in serial mode here except within the GDAL warped VRT machinery, which can use threads to parallelize warp source fetching and warping.